src/main/catalog-sources/kubeconfig-sync.ts

```diff
@@ -68,7 +74,7 @@ export class KubeconfigSyncManager extends Singleton {
       this.startNewSync(filePath);
     }
 
-    this.syncListDisposer = UserStore.getInstance().syncKubeconfigEntries.observe(change => {
+    this.syncListDisposer = UserStore.getInstance().syncKubeconfigEntries.observe_(change => {
```

Comment:
Why the `_`?

Reply:
There is no `.observe()` anymore in `mobx-6`

Reply:
Yes it does: https://mobx.js.org/intercept-and-observe.html

Just is a free function instead.

Reply:
Replaced.